src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToCopyLocalDependencies.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Xml.Linq;
+
+using FluentAssertions;
+
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToCopyLocalDependencies : SdkTest
```

Comment:
Add a test testing whether a library project gets package assets copied local by default.  As stated previously, I think the default should probably be false.  If that is the default, we should probably also have a test that you can turn it on.

Also consider some coverage of projects targeting .NET Standard.

Reply:
We should also have some test coverage for building self-contained apps.  Is the expectation that in that case all of the self-contained assets will be copied to the output folder too?  If so, then we should modify the self-contained publish tests to also test the same scenarios with build instead of publish.  If not, then we should add tests here that verify that runtime assets aren't copied to the output folder on self-contained build.

Reply:
I'll add a test for a library to ensure that the assets aren't being copied locally.

We are copying local all of the self-contained assets on build with these changes.  I'm not sure what the right call is in terms of user experience: either build and publish have the same contents or self-contained is purely a publish concept and we only copy those assets on build.  I'll add a test for this as well.

Given this is going into the first preview, my hope was that user feedback would help guide this.